src/Tools/dotnet-counters/Program.cs

```diff
@@ -54,6 +54,35 @@ internal class Program
                 new Option[] { },
                 handler: CommandHandler.Create<IConsole>(List));
 
+        private static Command ListProcessesCommand() =>
+            new Command(
+                "list-processes",
+                "Display a list of dotnet processes that can be monitored.",
+                new Option[] { },
+                handler: CommandHandler.Create<IConsole>(ListProcesses));
+
+        private static int ListProcesses(IConsole console)
+        {
+            try
+            {
+                var processes = EventPipeClient.ListAvailablePorts()
+                    .Select(System.Diagnostics.Process.GetProcessById)
+                    .Where(process => process != null)
+                    .OrderBy(process => process.ProcessName)
+                    .ThenBy(process => process.Id);
+
+                foreach (var process in processes)
+                    Console.Out.WriteLine($"{process.Id, 10} {process.ProcessName, -10} {process.MainModule.FileName}");
+
+                return 0;
+            }
+            catch (Exception ex)
```

Comment:
Because you're using `CommandLineBuilder.UseDefaults`, exception handling is already provided, including the same default return codes you're using here.